### PR TITLE
Added: definition of "split_settings" layout

### PR DESCRIPTION
**Pull request by Miexil**
### What does this PR do?
As discussed, adds the split_settings layout definition.

### Good PR checklist
- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [ ] Added/updated documentation
- [ ] Properly labeled

### CODE REVIEW

File: less/utilities/_base.less [hunk 1]
```diff
@@ -0,0 +1 @@
+.box-shadow { box-shadow: @box-shadow }
```
Comment (JulienVannier66) [inline less/utilities/_base.less:1]: Don't forget to add a new line at the end for file

Comment (phndiaye) [reply less/utilities/_base.less:1]: seems legit to me, don't know why Github is throwing a warning

**Review by phndiaye — APPROVED** [review]

File: less/utilities/_base.less (hunk 1 above)
Comment (Miexil) [reply less/utilities/_base.less]: Fixed

File: less/layouts/_split-settings.less [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    min-width: 600px;
+  }
+}
+
+// Will remove when proper documentation will be added.
+// Usage:
+// <div class="split-settings">
+//   <div class="split-settings__info"></div>
+//   <div class="split-settings__content"></div>
+// </div>
```
[the remark this answers is not in the document]
Comment (Miexil) [reply less/layouts/_split-settings.less]: Fixed

**Review by Hegazy360 — APPROVED** [review]

**Pull request merged**